### ISSUE TITLE
duplicates: how to know which one to remove

### ISSUE THREAD

**Issue by myrmoteras** [plazi/Biodiversity-Literature-Repository#54]
@gsautter in this case https://zenodo.org/communities/biosyslit/search?page=1&size=20&q=leidy&type=publication&sort=publication_date there are a number of articles that are duplicates or even triplates. How can we find out, which ones to remove, that is which ones are used as source for treatments?

**Comment by gsautter** [comment]
Could well be all of them ... and respective duplicates in the figures along the way. Uploading the same IMF twice appears to cause some problems in this regard at the moment, working on a solution as I type this.

**Comment by myrmoteras** [comment]
here another duplicate which might be caused now by the reprocessing of the Pensoft journals? see the upload dates

http://treatment.plazi.org/GgServer/html/F45D2CDE4E0F6595D0F2F95CD4821CA8 / http://treatment.plazi.org/GgServer/summary/E502FFD3FFE1B233FFA1FFC4FFB6FFAE (upload: (2019-03-21 01:04:50))

http://treatment.plazi.org/GgServer/html/893D33B430A6C8238BE483A2683D52A8 / http://treatment.plazi.org/GgServer/summary/FF871A4AFFBD9976FFF81F54B531FA0A upload (2015-12-27 22:55:30)

**Comment by gsautter** [comment]
Nope, not by the current reprocessing ... something we did in March 2019 (the time of the second import). I'm aware of this, and the re-import will resolve all of these cases. Just waiting for Teodor to add the Zenodo deposition numbers to the TaxPub.
